### PR TITLE
Update MS-DIAL FBMN doc for Waters MSe data processing

### PR DESCRIPTION
**Pull request by KyobinKang**

### CODE REVIEW

File: docs/featurebasedmolecularnetworking-with-ms-dial.md [hunk 1]
```diff
@@ -130,13 +136,13 @@ Go to Menu: Export / Alignment result
...
 
-Make sur the “GNPS export” box is checked, and set the spectra type to “centroid”.
+Make sure the “GNPS export” box is checked, and set the spectra type to “centroid”.
 
 Set the directory and file name, then click “Export”. As a result, you will acquire a MS/MS spectral summary and the feature quantification table.
 
 ![img](img/ms-dial/export_2.png)
 
-[Note] The GNPS export option of MS-DIAL will create a feature table based on MS1 peak height. If you want a feature table based on peak area, check the “Raw data matrix (Area)”.
+[Note] The GNPS export option of MS-DIAL will create a feature table based on MS1 peak height. If you want a feature table based on peak area, check the “Raw data matrix (Area)”.  The resulting file has a different structure from TXT file given by GNPS export, so you should reformat the area matrix file.
```
Comment (lfnothias) [inline docs/featurebasedmolecularnetworking-with-ms-dial.md:145]: Hey @KyobinKang, is it something that you observe only when MSe data is processed ? Or DDA as well ?

File: docs/featurebasedmolecularnetworking-with-ms-dial.md [hunk 2]
```diff
@@ -59,12 +59,16 @@ Go to Menu: File / New Project
...
 
+[Note] If you are processing with Waters MSe data, set MS method type to 'SWATH-MS or conventional All-ions method' and use [this experiment file](/static/MSe_experiment.txt).
+
 #### 2. Import Files
 
 Click “browse” and open the dropdown menu for file types, and select “mzML file(*.mzml)”. The other available options are .abf or .cdf files.
 
 [Note] Data files should be placed at the same folder with the project file.
 
+[Note] If you are processing with Waters MSe data, convert raw data files into .abf and import. You can download ABF converter [here](http://www.reifycs.com/AbfConverter/index.html)
```
Comment (lfnothias) [inline docs/featurebasedmolecularnetworking-with-ms-dial.md:70]: The . is missing at the end of the line

File: docs/featurebasedmolecularnetworking-with-ms-dial.md [hunk 3]
```diff
@@ -110,10 +114,12 @@ Click “Finish”, then MS-DIAL will perform all the feature detection and alig
 ![img](img/ms-dial/set-parameters-alignment.png)
 
 
-#### Processing Steps for MSE data (Experimental)
+#### Processing Steps for MSE data
 
 - See the [MS-DIAL documentation for MSE data processing](https://mtbinfo-team.github.io/mtbinfo.github.io/MS-DIAL/tutorial#chapter-8), and export the files as indicated below. If you encounter any issue, please contact the GNPS team on the [forum](https://groups.google.com/forum/#!forum/molecular_networking_bug_reports) or [GitHub](https://github.com/CCMS-UCSD/GNPS_Workflows/issues).
 
+- Here is an example MSe-based MS-DIAL networking [job](https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=283fc35908b44b49a82bb5b8371ae44d). MSe .Raw and converted ABF files can be downloaded from MassIVE accession no. MSV000084931.
```
Comment (lfnothias) [inline docs/featurebasedmolecularnetworking-with-ms-dial.md:121]: Please add the link to the MSV deposition: https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=e92f856f511042f487a8fea44955b581

File: docs/featurebasedmolecularnetworking-with-ms-dial.md (hunk 1 above)
Comment (KyobinKang) [reply docs/featurebasedmolecularnetworking-with-ms-dial.md:145]: Does not matter whether it's DDA or DIA. The GNPS export module of MS-DIAL gives a height-based table as a default. Maybe we should suggest a revision on this to Hiroshi?

File: docs/featurebasedmolecularnetworking-with-ms-dial.md (hunk 2 above)
Comment (KyobinKang) [reply docs/featurebasedmolecularnetworking-with-ms-dial.md]: Fixed

File: docs/featurebasedmolecularnetworking-with-ms-dial.md (hunk 3 above)
Comment (KyobinKang) [reply docs/featurebasedmolecularnetworking-with-ms-dial.md]: Thanks!

File: docs/featurebasedmolecularnetworking-with-ms-dial.md (hunk 1 above)
Comment (lfnothias) [reply docs/featurebasedmolecularnetworking-with-ms-dial.md:145]: Indeed, I think that would be nice to have such an option. Lets reach out to Hiroshi to see if that is possible.

**Comment by lfnothias** [comment]
@mwang87, that looks good to me.

**Pull request merged**